res/controllers/Hercules_DJControl_Jogvision-scripts.js

```diff
@@ -0,0 +1,284 @@
+///////////////////////////////////////////////////////////////////////////////
+// JSHint configuration                                                      //
+///////////////////////////////////////////////////////////////////////////////
+/* global engine                                                             */
+/* global print                                                              */
+/* global midi                                                               */
+/////////////////////////////////////////////////////////////////////////////// 
+// ****************************************************************************
+// * Mixxx mapping script file for the Hercules DJControl Jogvision.
+// * Author: DJ Phatso, contributions by Kerrick Staley and David TV
+// * Version 1.5 (January 2020)
+// * Version 1.4 (December 2019)
+// * Version 1.3 (November 2019)
+// * Version 1.2 (November 2019)
+// * Version 1.1 (March 2019)
+// * Forum: https://www.mixxx.org/forums/viewtopic.php?f=7&t=12580
+// * Wiki: https://www.mixxx.org/wiki/doku.php/hercules_dj_control_jogvision
+//
+// Changes to v1.5
+// - Fixed regression not updating the VuMeters
+// - Fixed regression not updating the CUE/MIX buttons
+//
+// Changes to v1.4
+// - Code cleanup and standarization
+// - Added "orage" track led color (ala Serato) when play position is beyond the
+//   part of the track, but before the "end of track" Mixxx signal
+//
+// Changes to v1.3
+// - Enabled the creation of beatloop (beatloop_activate) by using SHIFT+LoopON
+// - Changed "LOOP SIZE" to adjust the beatjump_size (you still can change size
+//   with surrounding buttons: reloop_toggle key)
+// - Added SHIFT+"LOOP SIZE" to move loop left or right by N beats (beatjump)
+//
+// Changes to v1.2
+// - Enabled Jogwheel Outer LED rotation
+// - Enabled Beat LEDs
+// 
+// Changes to v1.1
+// - Controller knob/slider values are queried on startup, so MIXXX is synced.
+// - Fixed vinyl button behavior the first time it's pressed.
+// 
+// v1.0 : Original release
+//
+// ****************************************************************************
+
+var on = 0x7F;
+var off = 0x00;
+var alpha = 1.0 / 8;
+var beta = alpha / 16;
+var ledRotationSpeed = 2;
+var ledRotationTimer = 0;
+var masterLeds = 0x90;
+
+var DJCJV = {};
+
+DJCJV.vinylModeActive = true;
+DJCJV.Channel = [];
+DJCJV.Channel['[Channel1]'] = { "central": 0x90, "deck": 0xB0, "beatPosition": 1, "rotation": 0x00, "n": 1 };
```

Comment:
You're using a mix of double and single quotes for strings. I'd prefer a consistent style and switch to double quotes only.

If you place [this `.eslintrc.json` file](https://github.com/mixxxdj/mixxx/blob/9cbc97f3cc51ceb6ebb2f9ca8cfd6ac8a12251da/.eslintrc.json) into the `mixxx` directory and run

    $ eslint --fix res/controllers/Hercules_DJControl_Jogvision-scripts.js

 it should replace all single with double quotes automatically.